src/appLoader.js

```diff
@@ -13,7 +13,6 @@ import Main from 'src/pages/Main';
 const rootElement = document.getElementById('root');
 
 RModal.defaultStyles = { overlay: {}, content: {} };
```

Comment:
There's still some React Modal configuration here in Terra UI because ModalDrawer also uses React Modal.